Sources/NIO/EventLoop.swift

```diff
@@ -1366,16 +938,19 @@ private final class ScheduledTask {
 }
 
 extension ScheduledTask: CustomStringConvertible {
+    @usableFromInline
     var description: String {
-        return "ScheduledTask(readyTime: \(self.readyTime))"
+        return "ScheduledTask(readyTime: \(self._readyTime))"
     }
 }
 
 extension ScheduledTask: Comparable {
+    @inlinable
```

Comment:
Why does this need to be `inlinable`? This object is purely internal and only ever compared in `PriorityQueue.push`, which should be automatically specialised inside NIO.

Reply:
semi-automatic ;). These days we don't really think about every single inlinable anymore but I can take it out if you want.